fission-server/src/routes/volume.rs

```diff
@@ -64,7 +64,12 @@ pub async fn update_cid(
 ) -> AppResult<(StatusCode, Json<NewVolumeRecord>)> {
     let mut conn = db::connect(&state.db_pool).await?;
     let account = Account::find_by_username(&mut conn, username).await?;
-    let volume = account.update_volume_cid(&mut conn, &payload.cid).await?;
+
+    let volume: NewVolumeRecord = if account.volume_id.is_some() {
```

Comment:
Does it make more sense to do this here, or to create the volume (possibly with a special genesis CID?) when the account is created?

Reply:
For now we want to do it this way. But I think there were plans to create a WNFS and another key pair for it on the server side. We still need to flesh that out though, not sure if taking more power away from the client is a good idea 🤔